### ISSUE TITLE
Show a unique gif on Every 10th Visitors

### ISSUE THREAD

**Issue by ZiaCodes** [ZiaCodes/counter.js#2]
Make a logic such that when the count will reach let say 10 or 100 or even 1000 then render a gif to congratulate the visitors.

Note : Before making pull request make sure you have done all the work and please add some necessary gifs and photos.

**Comment by Vipul2504** [comment]
I want to work on this issue under JWOC

**Comment by ZiaCodes** [comment]
> I want to work on this issue under JWOC

Great Good luck

**Comment by Vipul2504** [comment]
I have to render just one GIF locally or via API?

**Comment by ZiaCodes** [comment]
> I have to render just one GIF locally or via API?

Yes you have to render a gif for every 10th visitors . You can use any method you like , just make sure it is working

**Comment by r3yc0n1c** [comment]
if no one is working on this, I would like to work on this issue under JWOC

**Comment by ZiaCodes** [comment]
> if no one is working on this, I would like to work on this issue under JWOC

Yeah you can ..

**Comment by r3yc0n1c** [comment]
update: almost done! will merge the PR today
